### PR TITLE
Changes to include cleanup_env as a part of Jenkin

### PR DESCRIPTION
**Pull request by anrao19**
Signed-off-by: Anuchaithra Rao <anrao@redhat.com>
Jenkin file changes to include cleanup_env as a part of Jenkin

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

File: pipeline/rhos_quota/Jenkinsfile-cleanup-env.groovy [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                disableSubmodules: false,
+                parentCredentials: false,
+                recursiveSubmodules: true,
+                reference: '',
+                trackingSubmodules: false
+            ]],
+            submoduleCfg: [],
+            userRemoteConfigs: [[url: 'https://github.com/red-hat-storage/cephci.git']]
+        ])
+        script {
```
Comment (psathyan) [inline pipeline/rhos_quota/Jenkinsfile-cleanup-env.groovy:26]: FYI, we can avoid the script tag. It is more of a logical grouping of commands. However, it is useful for declarative method. Over here, we use scripted pipelines

File: pipeline/rhos_quota/Jenkinsfile-cleanup-env.groovy [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            sharedLib = load("${env.WORKSPACE}/pipeline/vars/common.groovy")
+            sharedLib.prepareNode()
+        }
+    }
+    stage('Cleanup environment and Send Email') {
```
Comment (psathyan) [inline pipeline/rhos_quota/Jenkinsfile-cleanup-env.groovy:31]: It is sufficient if we mention `Scrub RHOS-Ring`

**Review by psathyan — COMMENTED** [review]
Would it make sense to keep the folder name as `rhos_scripts` instead of `rhos_quota`?

@anrao19 @manasagowri what do you think?
Anuchaithra : Yes, I will may the changes

File: pipeline/rhos_quota/Jenkinsfile-cleanup-env.groovy (hunk 1 above)
Comment (anrao19) [reply pipeline/rhos_quota/Jenkinsfile-cleanup-env.groovy]: Sure @psathyan, Thanks for the info

**Review by psathyan — APPROVED** [review]

**Pull request merged**